website/content/docs/connect/proxies/envoy.mdx

```diff
@@ -93,28 +87,40 @@ responsibility for correctly configuring Envoy and ensuring version support etc.
 
 ## Intention Enforcement
 
-[Intentions] are enforced using Envoy's RBAC filters. Depending upon the
-configured [protocol] of the proxied service these are either enforced
-per-connection (L4) using a network filter or per-request (L7) using an HTTP
+[Intentions] are enforced using Envoy's RBAC filters. Depending on the
+configured [protocol] of the proxied service, intentions are either enforced
+per-connection (L4) using a network filter, or per-request (L7) using an HTTP
 filter.
 
 -> **Note:** Prior to Consul 1.9.0 intentions were exclusively enforced
 per-connection (L4) using an `ext_authz` network filter.
 
+## Fetching Certificates
+
+Envoy will use the [`CONSUL_HTTP_TOKEN`](/commands#consul_http_token) and [`CONSUL_HTTP_ADDR`](/commands#consul_http_addr) environment variables to contact Consul to fetch certificates if the following conditions are met:
+
+- The `CONSUL_HTTP_TOKEN` environment variable contains a Consul ACL.
+- The Consul ACL has the necessary permissions to read configuration for that service.
+
+If TLS is enabled on Consul, you will also need to add the following environment variables _prior_ to starting Envoy:
+
+- [`CONSUL_CACERT`](/commands#consul_cacert)
+- [`CONSUL_CLIENT_CERT`](/commands#consul_client_cert)
+- [`CONSUL_CLIENT_KEY`](/commands#consul_client_key)
```

Comment:
According to the linked issue, the following parameters are also required.

```suggestion
- [`CONSUL_CLIENT_KEY`](/commands#consul_client_key)
- [`CONSUL_HTTP_SSL`](/commands#consul_http_ssl)
```

The issue also suggests that `CONSUL_HTTP_ADDR` and `CONSUL_GRPC_ADDR` be defined with the scheme set to `https://`, but according to the [`CONSUL_GRPC_ADDR`](https://www.consul.io/commands#consul_grpc_addr) docs it should be sufficient just to set `CONSUL_HTTP_SSL=true`.